bitnami/airflow/templates/_init_containers_sidecars.tpl

```diff
@@ -222,11 +230,20 @@ Returns an init-container that waits for db migrations to be ready
       . /opt/bitnami/scripts/airflow-env.sh
       . /opt/bitnami/scripts/libairflow.sh
 
+      info "Trying to connect to the database server"
+      airflow_wait_for_db_connection
```

Comment:
Isn't this step redundant? I mean, because you do `airflow_wait_for_db_migrations` right after.

Reply:
The `wait_for_db_connection` checks you can connect with the database from the pod. The `wait_for_db_migrations` check the database migrations have finished. The idea is to not spend `wait_db_migrations` cycles while the database is getting started and the pod can't connect with it.